src/test/bus.test.js

```diff
@@ -62,9 +62,9 @@ describe('Buses', () => {
   it('shouldn\'t get bus by id if the bus doesn\'t exist in the system', (done) => {
     chai
       .request(app)
-      .get('/api/v1/buses/2')
+      .get('/api/v1/buses/100')
       .set('x-access-token', adminToken)
-      .set('request.params', 2)
+      .set('request.params', 100)
       .end((err, res) => {
         expect(res).to.have.status(401);
         expect(res).to.have.property('status');
```

Comment:
The second assertion is extraneous. Also, it'd be better to add another assertion.